src/containers/Transactions/simpleTab.scss

```diff
@@ -1,14 +1,15 @@
 @import '../shared/css/variables';
 
-.simple-body {
+.simple-body-tx {
   .rows {
     width: 100%;
     padding-right: 0px;
     padding-bottom: 40px;
+    padding-left: 16px;
     float: left;
 
     @include for-size(desktop-up) {
-      width: calc(100% - 276px - 32px);
+      width: calc(100% - 276px - 48px);
```

Comment:
i don't love these magic numbers. is there no way to define this relatively using percents and flex?

Reply:
There might be. I believe the idea is that you have the 276px for the sidebar with ledger index and stuff and 48px for buffer, and everything else should be taken up by this section. I'm not sure how else to put it, but I'm all ears.